src/generators/python/renderers/ClassRenderer.ts

```diff
@@ -83,7 +83,8 @@ export const PYTHON_DEFAULT_CLASS_PRESET: ClassPresetType<PythonOptions> = {
     if (Object.keys(properties).length > 0) {
       const assigments = Object.values(properties).map((property) => {
         if (!property.required) {
-          return `if hasattr(input, '${property.propertyName}'):\n\tself._${property.propertyName} = input.${property.propertyName}`;
+          return `if hasattr(input, '${property.propertyName}'):
+  self._${property.propertyName} = input.${property.propertyName}`;
```

Comment:
Adding the newline was correct, but you need to use 2 spaces after it instead of tab. Python can take any indents with spaces, but need to be consistent within the same file. It's not quite clear here if there are spaces before the `self._...` concatenation.